libs/types-common/src/Data/Json/Util.hs

```diff
@@ -34,12 +43,20 @@ infixr 5 #
 -- | A newtype wrapper for 'UTCTime' that formats timestamps in JSON with
 -- millisecond precision instead of the default picosecond precision.
 newtype UTCTimeMillis = UTCTimeMillis UTCTime
+  deriving (Eq)
+
+instance Show UTCTimeMillis where
+    showsPrec d t = showParen (d > 10) . showString $ "UTCTimeMillis " ++ showUTCTimeMillis t
```

Comment:
Think we can remove `"UTCTimeMillis "` from `Show`

Reply:
done